plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -1029,6 +1030,14 @@ read_only::get_table_rows_result read_only::get_table_rows( const read_only::get
             return f128;
          });
       }
+      else if (p.key_type == "sha256") {
+          return get_table_rows_by_seckey<index256_index, checksum256_type>(p, abi, [&p](const checksum256_type& v)->key256_t {
```

Comment:
`&p` not used in lambda

Reply:
yes, left over from earlier code I used it for logging, will remove it.

`i256` assumes the string is in binary 32 bytes numeric string **not in hex**  , but also when I converted sha256 hex 64 bytes key to binary 32 and used it with `i256` it did not find my rows.

Reply:
I will test it again with `i256` and let you know @heifner , thank you!

Reply:
`python -c 'print str(int("c01b39c7a35ccc3b081a3e83d2c71fa9a767ebfeb45c69f08e17dfe3ef375a7b", 16))'
`
**86892170580511395102392104225515491945535356631148603271263139179137605261947**

Reply:
@heifner , I will use that decimal again to test if `i256` works .